main/src/test/java/org/mobilitydata/gtfsvalidator/cli/CliParametersAnalyzerTest.java

```diff
@@ -17,130 +17,123 @@
 package org.mobilitydata.gtfsvalidator.cli;
 
 import static com.google.common.truth.Truth.assertThat;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyNoMoreInteractions;
-import static org.mockito.Mockito.when;
 
-import java.util.logging.Handler;
-import java.util.logging.LogRecord;
-import java.util.logging.Logger;
-import org.junit.Before;
+import com.beust.jcommander.JCommander;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
-import org.mockito.ArgumentCaptor;
-import org.mockito.Mockito;
 
 @RunWith(JUnit4.class)
 public class CliParametersAnalyzerTest {
-  private Handler mockHandler = null;
-  private ArgumentCaptor<LogRecord> logRecordCaptor = null;
+  @Test
+  public void isValid_false() {
```

Comment:
Please split that into separate tests for each scenario.

Remember that a failed assertThat interrupts your test. In your code, you test multiple scenarios, so the first failed scenario prevents the following ones and test output is incomplete.

Reply:
Please avoid code duplication. Create a helper static method that would accept your String [] argv, validate them and return the result of isValid().